### PR TITLE
Add password inputs to password reset edit form

### PR DESCRIPTION
**Pull request by joseph-lozano**
the reset_password edit form did not have password or password_confirmation inputs.

### CODE REVIEW

**Comment by joseph-lozano** [comment]
Happy to add assertions that these inputs exist, but I also am not sure they are necessary.

File: priv/templates/phx.gen.auth/reset_password_edit.html.heex [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+    <:subtitle>Use the form below to reset your password</:subtitle>
   </.header>
 
   <.simple_form :let={f} for={@changeset} action={~p"<%= schema.route_prefix %>/reset_password/#{@token}"}>
     <.error :if={@changeset.action}>
       Oops, something went wrong! Please check the errors below.
     </.error>
 
-    <.input field={{f, :email}} type="email" placeholder="Email" required />
+    <.input field={{f, :password}} type="password" placeholder="Password" required />
```
Comment (bemesa21) [inline priv/templates/phx.gen.auth/reset_password_edit.html.heex:12]:
```suggestion
    <.input field={{f, :password}} type="password" label="New Password" required />
```

File: priv/templates/phx.gen.auth/reset_password_edit.html.heex [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+    <.input field={{f, :password_confirmation}} type="password" placeholder="Password Confirmation" required />
```
Comment (bemesa21) [inline priv/templates/phx.gen.auth/reset_password_edit.html.heex:13]:
```suggestion
    <.input field={{f, :password_confirmation}} type="password" label="Confirm new password" required />
```

File: priv/templates/phx.gen.auth/reset_password_edit.html.heex [hunk 3]
```diff
@@ -1,19 +1,21 @@
 <div class="mx-auto max-w-sm">
   <.header class="text-center">
-    Forgot your password?
-    <:subtitle>We'll send a password reset link to your inbox</:subtitle>
+    Password Reset
+    <:subtitle>Use the form below to reset your password</:subtitle>
```
Comment (bemesa21) [inline priv/templates/phx.gen.auth/reset_password_edit.html.heex:4]:
```suggestion
```

File: priv/templates/phx.gen.auth/reset_password_edit.html.heex [hunk 4]
```diff
@@ -1,19 +1,21 @@
 <div class="mx-auto max-w-sm">
   <.header class="text-center">
-    Forgot your password?
-    <:subtitle>We'll send a password reset link to your inbox</:subtitle>
+    Password Reset
```
Comment (bemesa21) [inline priv/templates/phx.gen.auth/reset_password_edit.html.heex:3]: Reset password

**Review by bemesa21 — CHANGES_REQUESTED** [review]
Good catch! those inputs were definitely needed.

I suggested a few changes just to keep the template the same as the one generated with the --live option.

File: priv/templates/phx.gen.auth/reset_password_edit.html.heex (hunk 4 above)
Comment (bemesa21) [inline priv/templates/phx.gen.auth/reset_password_edit.html.heex:3]:
```suggestion
    Reset Password
```

**Review by bemesa21 — CHANGES_REQUESTED** [review]

**Comment by bemesa21** [comment]
✨✨✨💫

**Pull request merged**